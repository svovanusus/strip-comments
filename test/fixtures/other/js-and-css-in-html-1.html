<script>
  /**
   * Some Script
   * By Someone
  */
  console.log(1); // Comment
</script>

<style>
/* This is style 2 */
.class {
  background: url(https://some.site/url/file.jpg);
  /*padding: 5px;*/
}
</style>

<div style="display: none;">
  <p>Hidden block</p>
</div>
