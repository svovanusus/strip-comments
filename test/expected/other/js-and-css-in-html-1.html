<script>
    console.log(1); 
</script>

<style>
.class {
  background: url(https://some.site/url/file.jpg);
  }
</style>

<div style="display: none;">
  <p>Hidden block</p>
</div>
